duckduckgo/macos-browser
# CSV import from Bitwarden

Task/Issue URL: https://app.asana.com/0/1177771139624306/1204289821417375/f

**Description**:
CSV import from Bitwarden fixed by adding more possible column names

**Steps to test this PR**:
1. Export CSV from Bitwarden or use the file attached in Asana task
2. Import the file
3. Make sure credentials are imported correctly



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
